Dependency on zip extension

Chicken/egg raises its head when installing for example in a Dockerfile; for example, given the following:

```Dockerfile
FROM php:8.4-cli

COPY --from=ghcr.io/php/pie:bin /pie /usr/bin/pie

RUN pie install xdebug/xdebug
```

It would be useful if this worked out the box, but it does not, as Box correctly identifies that `zip` ext is missing when you try to build it, `docker build --file test2.Dockerfile .` for example:

```
 > [stage-0 3/3] RUN pie install xdebug/xdebug:                                                                                             
0.204                                                                                                                                       
0.204 Box Requirements Checker                                                                                                              
0.204 ========================
0.204 
0.204 > Using PHP 8.4.2
0.204 > PHP is not using any php.ini file.
0.204 
0.204 > Checking Box requirements:
0.204   ...E.
0.204 
0.204                                                                                 
0.204  [ERROR] Your system is not ready to run the application.                       
0.204                                                                                 
0.204 
0.204 Fix the following mandatory requirements:
0.204 =========================================
0.204 
0.204  * This application requires the extension "zip". You either need to enable it
0.204    or request the application to be shipped with a polyfill for this extension.
0.204 
------
test2.Dockerfile:5
--------------------
   3 |     COPY --from=ghcr.io/php/pie:bin /pie /usr/bin/pie
   4 |     
   5 | >>> RUN pie install xdebug/xdebug
   6 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c pie install xdebug/xdebug" did not complete successfully: exit code: 1
```

--- a/composer.json
+++ b/composer.json
@@ -28,7 +28,6 @@
     ],
     "require": {
         "php": "8.1.*||8.2.*||8.3.*||8.4.*",
-        "ext-zip": "*",
         "composer/composer": "^2.8.4",
         "composer/pcre": "^3.3.2",
         "composer/semver": "^3.4.3",

--- a/docs/usage.md
+++ b/docs/usage.md
@@ -40,6 +40,31 @@ Instead of `bin` tag (which represents latest binary-only image) you can also us
 > [!IMPORTANT]  
 > Binary-only images don't include PHP runtime so you can't use them for _running_ PIE. This is just an alternative way of distributing PHAR file, you still need to satisfy PIE's runtime requirements on your own.
 
+#### Example of PIE working in a Dockerfile
+
+This is an example of how PIE could be used to install an extension inside a
+Docker image. Note that, like Composer, you need something like `unzip`, the
+[Zip](https://www.php.net/manual/en/book.zip.php) extension, or `git` to be
+installed.
+
+```Dockerfile
+FROM php:8.4-cli
+
+# Add the `unzip` package which PIE uses to extract .zip files
+RUN apt-get -y update \
+    && apt-get install -y --no-install-recommends unzip \
+    && rm -rf /var/lib/apt/lists/*
+
+# Copy the pie.phar from the latest `:bin` release
+COPY --from=ghcr.io/php/pie:bin /pie /usr/bin/pie
+
+# Use PIE to install an extension...
+RUN pie install asgrim/example-pie-extension
+```
+
+If the extension you would like to install needs additional libraries or other
+dependencies, then these must be installed beforehand too.
+
 ## Prerequisites for PIE
 
 Running PIE requires PHP 8.1 or newer. However, you may still use PIE to install

--- a/docs/usage.md
+++ b/docs/usage.md
@@ -51,9 +51,10 @@ installed.
 FROM php:8.4-cli
 
 # Add the `unzip` package which PIE uses to extract .zip files
-RUN apt-get -y update \
-    && apt-get install -y --no-install-recommends unzip \
-    && rm -rf /var/lib/apt/lists/*
+RUN export DEBIAN_FRONTEND="noninteractive"; \
+    set -eux; \
+    apt-get update; apt-get install -y --no-install-recommends unzip; \
+    rm -rf /var/lib/apt/lists/*
 
 # Copy the pie.phar from the latest `:bin` release
 COPY --from=ghcr.io/php/pie:bin /pie /usr/bin/pie

